src/documentation/transxchange_read.md

```diff
@@ -91,14 +103,25 @@ rail | Train | Train | Train | Train
 tram | Tramway | Tramway| Tramway| Tramway
 trolleyBus | Shuttle | Shuttle | Shuttle | Shuttle
 
+If *Services/Service/Mode* is not specified or the value is unknown (a different value than those listed above), the default mode `Bus` is used.
+
 ### routes.txt
-A Route is created from a line and a direction of journey patterns (*StandardService/JourneyPattern/Direction*/).
-Routes might not be specified at all in the xml --> reconstruct routes after trips?
+A Route is created from a line and a direction of journey patterns (*StandardService/JourneyPattern/Direction*).
+
+NTFS field | TransXChange element | Mapping rule/Comment
+--- | --- | ---
+route_id | *Services/Service/ServiceCode*, *Services/Service/Line{id}* *StandardService/JourneyPattern/Direction* | This field is prefixed and formed by the concatenation of *Services/Service/ServiceCode*, *Services/Service/Line{id}* and *StandardService/JourneyPattern/Direction* separated by a `:`. Ex. "\<prefix>:1_58_BC:SL1:inbound".
+route_name |  | "[first stop of the first trip] - [last stop of the first trip]" (1)
+direction_type | *StandardService/JourneyPattern/Direction* | The value is set to `inbound` or `clockwise` when the specified value for the direction is `inboundAndOutbound` or `circular`, respectively.
+line_id | *Services/Service/ServiceCode*, *Services/Service/Line{id}* | This field is prefixed. Link to the file [lines.txt](#linestxt).
+destination_id |  | `stop_id` of the stop_area of the last stop of the first trip (1). Link to the file [stops.txt](#stopstxt).
+
+(1) The first trip (in alphabetical order) of a route is the one with the smallest `trip_id` value.
 
 ### calendar_dates.txt
 The validity period of a service is stated in *Services/Service/OperatingPeriod*. In case the validity period is open ended (the *EndDate* is not specified), the default value [*StartDate* + 180 days] should be used.
```

Comment:
Some of the data are showing `EndDate` like `9999-01-01`. The proposition in this case is to limit to `StartDate + 2 year`.

Note: It is mostly a technical reason due to the fact that the connector is creating a list of all the dates.

Reply:
As the `StartDate` could be far in the past, we cannot use a `StartDate + 2Y`. Instead, will add a mandatory parameter specifying the EndDate of the loaded data to specify (or truncate) calendars.